style.css

```
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+
+  --main-color: "#172b4d";
+  --secondary-color: #36b37e;
+}

```

- [ ] OPTIONAL please don't declare this variables for every element. simply add then in a `:root` section. 👍
something close to this could be much better I believe.
```suggestion
}

:root {
  --main-color: "#172b4d";
  --secondary-color: #36b37e;
}

```

---

changed